docs/TESTCAFE.md

```diff
@@ -47,3 +71,21 @@ Required to connect to your BrowserStack account. Get this information from your
 
 #### `BROWSERSTACK_PROJECT_NAME`
 Name of the BrowserStack project. Set to `plio`. It allows to name the build, which is useful for keeping track of test runs on the BrowserStack dashboard.
+
+#### `GOOGLE_OAUTH2_CLIENT_ID`
+Google refresh token retrieved in the pre-requisites.
+
+#### `GOOGLE_OAUTH2_CLIENT_SECRET`
+Google client ID retrieved in the pre-requisites.
+
+#### `GOOGLE_OAUTH2_REFRESH_TOKEN`
+Google client secret retrieved in the pre-requisites.
+
+#### `BACKEND_CONVERT_SOCIAL_AUTH_TOKEN_URL`
+The convert token URL in the Plio backend repo that converts the Social Auth Token and returns a valid Auth Token. By default, the value is http://0.0.0.0:8001/auth/convert-token.
+
+#### `BACKEND_DEFAULT_OAUTH2_CLIENT_ID`
+The client id for Plio backend application.
+
+#### `BACKEND_DEFAULT_OAUTH2_CLIENT_SECRET`
```

Comment:
BACKEND_SECRET_KEY seems to be missing!

Reply:
DJANGO_DEFAULT_OAUTH2_CLIENT_SETUP based mentioned in CYPRESS.md - don't we need it `BACKEND_` equivalent?

Reply:
@dalmia can't find `DJANGO_DEFAULT_OAUTH2_CLIENT_SETUP` anywhere. Where do you see it? It's already converted to it's `BACKEND_` equivalent I believe.